src/commands/indexer/index.ts

```diff
@@ -359,7 +358,29 @@ export default class Indexer extends HealthCheck {
               ]!.bloomEvent.decode<TransferERC721Event>(type, interestingTransaction.log!)
               //            log('transferERC721Event', transferERC721Event);
               if (transferERC721Event !== null) {
-                await this.handleTransferERC721Event(job, interestingTransaction, transferERC721Event, tags)
+                let isNewMint = false
```

Comment:
@BravoNatalie I think having this logic in the file `handle-transfer-erc721-event.ts` file. That way this switch case can get smaller and clearer.